specification/indices/put_template/IndicesPutTemplateRequest.ts

```diff
@@ -31,7 +31,7 @@ import { Duration } from '@_types/Time'
  * Index templates define settings, mappings, and aliases that can be applied automatically to new indices.
  * @rest_spec_name indices.put_template
  * @availability stack stability=stable
- * @availability serverless stability=stable visibility=public
+ * @availability serverless stability=stable visibility=private
```

Comment:
For consistency with the other legacy template APIs, can we simply remove the serverless line?

```suggestion
```